tests/render/renderer/test_suite_scaffold_notebook_renderer_pre_v013.py

```diff
@@ -189,10 +84,22 @@ def test_notebook_execution_with_pandas_backend(
     context = DataContext(root_dir)
     obs_validation_result = context.get_validation_result(suite_name)
     assert obs_validation_result.statistics == {
-        "evaluated_expectations": 3,
-        "successful_expectations": 3,
+        "evaluated_expectations": 2,
+        "successful_expectations": 2,
```

Comment:
Just to confirm @talagluck , this `3` --> `2` is because snapshot test has been removed?

Reply:
Hi Will - thanks for checking in on this! The 3 --> 2 is because the new profiler has one fewer table expectation than the old one (it removes `expect_table_column_count_to_equal` because that didn't seem necessary in light of having `expect_table_columns_to_match_ordered_list`).